[variables.md] expanding on recommendations for conditional variable (#16465)
DataDog/documentation: content/en/monitors/notify/variables.md

{{< /tabs >}}

> <div class="alert alert-info"><strong>Note</strong>: Any text or notification handle placed outside of the configured conditional variables are invoked with every monitor state transition. Any text or notification handle placed inside of configured conditional variables are only invoked with that specific monitor state transition.
> </br></br>
> If you configure a conditional block for a state transition into `alert` or `warning` condition with an **@-notifications** handle, it is recommended to configure a corresponding `recovery` condition in order for a recovery notification to be sent to the handle.
> </div>

Reviewer: ```suggestion
If you configure a conditional block for a state transition into `alert` or `warning` conditions with an **@-notifications** handle, it is recommended to configure a corresponding `recovery` condition in order for a recovery notification to be sent to the handle.

**Note**: Any text or notification handle placed **outside** the configured conditional variables is invoked with every monitor state transition. Any text or notification handle placed **inside** of configured conditional variables is only invoked if the monitor state transition matches its condition.
```

Author: cool -- that works for me!  thanks for the notes and feeback
